webinars/urls.py

```diff
@@ -1,10 +1,12 @@
 from django.urls import include, path
 from rest_framework import routers
 from . import views
+from .search import search
 
 router = routers.SimpleRouter()
 router.register(r'', views.WebinarViewSet, basename='Webinars')
 
 urlpatterns = [
+    path('search/', search, name='search'),
```

Comment:
I don't think two urls should have the same `name`. Might be better to name this something like `webinar_search`.
(The blog search url has the name `search`: https://github.com/openstax/openstax-cms/blob/main/openstax/urls.py#L39)

Reply:
Good call. Made me look up what the name parameter is for. Changed to webinar_search